docs/rfc/rfc-1.adoc

```diff
@@ -281,15 +281,22 @@ it.
 [[heartbeat]]
 === Heartbeats
 
+Governable Parameters:
+
+- `failed_heartbeat_sortition_timeout`: The amount of time an operator is
+  removed from the sortition pool after failing a heartbeat.
+- `heartbeat`: The number of group members required for a heartbeat to successful.
+
 To make sure that older wallets are still accessible for redemption, we need to
-perform heartbeats. The signing group signs each bitcoin block and then does _not_
-publish the result. If a signer suspects other signers are not online, they can
-issue an on-chain challenge to publish a specified signed bitcoin block (with a
-maximum block age). Since publishing this information costs the signers gas, the
-challenger must pay a deposit to be distributed to the signers if they pass.
-
-Any signer unable to publish the signed block within a specified amount of time
-will begin to be slashed and the challenger will be rewarded.
+perform heartbeats. The signing group signs every 40th ethereum block and then
+does _not_ publish the result. If there are ever less than `heartbeat`
+operators that participate in the heartbeat, the ones that did can create and
+sign a new transaction that lists the _inactive_ operators. Once this
+transaction is posted to the ethereum chain, we can iterate through the
+inactive operators, remove them from the sortition pool for
+`failed_heartbeat_sortition_timeout` amount of time (which disables their `T`
+rewards), transfer the from the wallet to another random wallet and close this
+wallet.
```

Comment:
See this discussion please: https://github.com/keep-network/sortition-pools/pull/130#pullrequestreview-797630853. We can't remove them from sortition pool because there is a chance the pool is locked at that moment. What we probably want to do instead, is to remove them from rewards.

Reply:
addressed in 3920f67